.github/workflows/pr_tests.yml

```diff
@@ -42,13 +39,12 @@ env:
   SNOWFLAKE_TEST_WAREHOUSE: ${{ secrets.SNOWFLAKE_TEST_WAREHOUSE }}
 
   # Postgres Connection
-  POSTGRES_TEST_HOST: ${{ secrets.POSTGRES_TEST_HOST }}
   POSTGRES_TEST_USER: ${{ secrets.POSTGRES_TEST_USER }}
   POSTGRES_TEST_PASS: ${{ secrets.POSTGRES_TEST_PASS }}
-  POSTGRES_TEST_PORT: ${{ secrets.POSTGRES_TEST_PORT }}
   POSTGRES_TEST_DBNAME: ${{ secrets.POSTGRES_TEST_DBNAME }}
+  POSTGRES_TEST_HOST: ${{ secrets.POSTGRES_TEST_HOST }}
```

Comment:
Not relevant for this line but can we change the base to a release branch?

Reply:
Done